src/components/OperatorArea/OperatorArea.tsx

```diff
@@ -33,7 +33,14 @@ export const OperatorArea: FunctionComponent<OperatorAreaProps> = ({
 
     if (date) {
       const publishDate = new Date(date);
-      items.push(`Published: ${format(publishDate, "MMMM dd, yyyy")}`);
+      items.push(
```

Comment:
We should eventually implement a simple `<Time />` component so we can make this consistent!